Psychtoolbox/PsychHardware/EyelinkToolbox/EyelinkDemos/SR-ResearchDemos/SimpleVideo/EyeLink_SimpleVideo.m

```diff
@@ -0,0 +1,396 @@
+    Eyelink('Command', 'add_file_preamble_text "%s"', preambleText);
+    
+    
+    %% STEP 2: SELECT AVAILABLE SAMPLE/EVENT DATA
+    % See EyeLinkProgrammers Guide manual > Useful EyeLink Commands > File Data Control & Link Data Control
+    
+    % Select which events are saved in the EDF file. Include everything just in case
+    Eyelink('Command', 'file_event_filter = LEFT,RIGHT,FIXATION,SACCADE,BLINK,MESSAGE,BUTTON,INPUT');
+    % Select which events are available online for gaze-contingent experiments. Include everything just in case
+    Eyelink('Command', 'link_event_filter = LEFT,RIGHT,FIXATION,SACCADE,BLINK,BUTTON,FIXUPDATE,INPUT');
+    % Select which sample data is saved in EDF file or available online. Include everything just in case
+    if ELsoftwareVersion > 3  % Check tracker version and include 'HTARGET' to save head target sticker data for supported eye trackers
+        Eyelink('Command', 'file_sample_data  = LEFT,RIGHT,GAZE,HREF,RAW,AREA,HTARGET,GAZERES,BUTTON,STATUS,INPUT');
+        Eyelink('Command', 'link_sample_data  = LEFT,RIGHT,GAZE,GAZERES,AREA,HTARGET,STATUS,INPUT');
+    else
+        Eyelink('Command', 'file_sample_data  = LEFT,RIGHT,GAZE,HREF,RAW,AREA,GAZERES,BUTTON,STATUS,INPUT');
+        Eyelink('Command', 'link_sample_data  = LEFT,RIGHT,GAZE,GAZERES,AREA,STATUS,INPUT');
+    end
+    
+    
+    %% STEP 3: OPEN GRAPHICS WINDOW
+    
+    % Open experiment graphics on the specified screen
+    if isempty(screenNumber)
+        screenNumber = max(Screen('Screens')); % Use default screen if none specified
+    end
+    window = Screen('OpenWindow', screenNumber, [128 128 128]); % Open graphics window
+    Screen('Flip', window);
+    % Return width and height of the graphics window/screen in pixels
+    [width, height] = Screen('WindowSize', window);
+    
+    
+    %% STEP 4: SET CALIBRATION SCREEN COLOURS/SOUNDS; PROVIDE WINDOW SIZE TO EYELINK HOST & DATAVIEWER; SET CALIBRATION PARAMETERS; CALIBRATE
+    
+    % Provide EyeLink with some defaults, which are returned in the structure "el".
+    el = EyelinkInitDefaults(window);
+    % set calibration/validation/drift-check(or drift-correct) background color. 
+    % It is important that this background colour is similar to that of the stimuli to prevent large luminance-based 
+    % pupil size changes (which can cause a drift in the eye movement data)
+    el.backgroundcolour = [115 115 115];% RGB grey
+    % set "Camera Setup" instructions text colour so it is different from background colour
+    el.msgfontcolour = [0 0 0];% RGB black
+    
+    % Set calibration beeps (0 = sound off, 1 = sound on)
+    % Setting beeps to off (0) for video targets
+    el.targetbeep = 0;  % sound a beep when a target is presented
+    el.feedbackbeep = 0;  % sound a beep after calibration or drift check/correction
+    
+    % Required for macOS Catalina users (/w PTB 3.0.17.11) to disable audio

```

Comment:
Is this still a thing for current PTB and GStreamer versions?

Reply:
can remove in future revisions. realized I previously interpreted this as being related to el.targetbeep and el.feedbackbeep.